package.json

```diff
@@ -76,6 +76,7 @@
     "focus-visible": "^5.2.0",
     "glider-js": "^1.7.6",
     "history": "^5.0.0",
+    "html-to-image": "github:merapi/html-to-image",
```

Comment:
Could you also create a PR with the fixes to the `html-to-image` repo? (If the changes are final now)

Reply:
I've created PRs on the [original repo](https://github.com/bubkoo/html-to-image/pulls), we will see how it goes, they are rather slow to respond.